pages/home/home.ejs

```diff
@@ -2,108 +2,512 @@
+          <%
+            var bannerRowClasses = 'row align-items-center';
+            if (isAuthenticated) {
+              bannerRowClasses += ' pt-4';
+            }
+          %>
+          <div class="<%= bannerRowClasses %>">
+            <%
+              var bannerColClasses = 'col-md-6';
+              if (isAuthenticated) {
+                bannerColClasses += ' my-1';
+              } else {
+                bannerColClasses += ' my-5';
+              }
+            %>
+            <div class="<%= bannerColClasses %>">
+              <%
+                var bannerH1Classes;
+                if (isAuthenticated) {
+                  bannerH1Classes = 'h2';
+                } else {
+                  bannerH1Classes = 'display-4 font-weight-normal';
+                }
+              %>
+              <h1 class="<%= bannerH1Classes %>">PrairieLearn</h1>
+              <%
+                var bannerLeadClasses = '';
+                if (!isAuthenticated) {
+                  bannerLeadClasses = 'lead font-weight-normal';
+                }
+              %>
+              <p class="<%= bannerLeadClasses %>">
+                <i class="fa fa-sync-alt fa-fw" aria-ignore="true"></i>
+                Mastery learning meets online assessment
+              </p>
+            </div>
+            <div class="col-md-6">
+              <div class="list-group">
+                <% if (!isAuthenticated || isStudent) { %>
+                <div class="list-group-item py-3 px-4">
+                  <h2 class="h6 d-inline font-weight-bold">Students</h2>
+                  <p class="d-inline text-muted text-right">&mdash; Master course content through randomized, spaced
+                    repetition</p>
+                  <p class="mt-2 mb-0">
+                    <% if (!isAuthenticated) { %>
+                    <a href="<%= plainUrlPrefix %>/login" class="btn btn-primary mb-1">
+                      <i class="fa fa-fw fa-user-graduate" aria-hidden="true"></i>
+                      Log in
+                    </a>
+                    <% } %>
+                    <a href="<%= plainUrlPrefix %>/enroll" class="btn btn-<% if (!isAuthenticated) { %>outline-<% } %>primary mb-1">
+                      <i class="fa fa-fw fa-edit" aria-hidden="true"></i>
+                      <% if (isAuthenticated) { %>
+                      Add or remove courses
+                      <% } else { %>
+                      Add course
+                      <% } %>
+                    </a>
+                  </p>
+                </div> <!-- list-group-item -->
+                <% } %> <!-- role === 'Student' -->
+                <% if (!isAuthenticated || (isAdministrator || isInstructor)) { %>
+                <div class="list-group-item py-3 px-4">
+                  <h2 class="h6 d-inline font-weight-bold">Instructors</h2>
+                  <p class="d-inline text-muted text-right">&mdash; Build autograded assessments using premade widgets and
```

Comment:
Question components are described as "elements" not widgets in the documentation.

```suggestion
                  <p class="d-inline text-muted text-right">&mdash; Build autograded assessments using premade elements and
```